app/components/ImagesWrapper/ImagesWrapper.tsx

```diff
@@ -11,20 +12,24 @@ export const ImagesWrapper = () => {
     <div className={styles.component}>
       <div className={styles.imagesContainer}>
         <div className={styles.imageWrapper}>
-          <a href="https://www.ag-exchange.org/" target="_blank">
+          <a href="https://www.emeraldcoastrealestatetips.com/" target="_blank">
             <Image
               className={styles.image}
-              src={firstImageSrc}
+              src={secondImageSrc}
               alt="Image"
               fill
             />
           </a>
         </div>
-        <div className={`${styles.imageWrapper} ${styles.imageWrapperText}`}>
-          <p>
-            Here can be your next <br />
-            <strong>idea.</strong>
-          </p>
+        <div className={styles.imageWrapper}>
+          <a href="https://www.emeraldcoastrealestatetips.com/" target="_blank">
```

Comment:
```suggestion
          <a href="https://www.ag-exchange.org/" target="_blank">
```

I guess here should be`ag-exchange` if you moved cards